addons/sale/security/ir.model.access.csv

```diff
@@ -17,27 +17,16 @@ access_res_partner_sale_manager,res.partner.sale.manager,base.model_res_partner,
 
 access_mail_activity_type_sale_manager,mail.activity.type.sale.manager,mail.model_mail_activity_type,sales_team.group_sale_manager,1,1,1,1
 
-access_product_category_sale_manager,product.category salemanager,product.model_product_category,sales_team.group_sale_manager,1,1,1,1
 access_product_pricelist_sale_user,product.pricelist.sale.user,product.model_product_pricelist,sales_team.group_sale_salesman,1,0,0,0
 access_product_pricelist_sale_manager,product.pricelist salemanager,product.model_product_pricelist,sales_team.group_sale_manager,1,1,1,1
 access_product_pricelist_item_sale_manager,product.pricelist.item salemanager,product.model_product_pricelist_item,sales_team.group_sale_manager,1,1,1,1
-access_product_attribute_sale_manager,product.attribute manager,product.model_product_attribute,sales_team.group_sale_manager,1,1,1,1
-access_product_attribute_value_sale_manager,product.attribute manager value,product.model_product_attribute_value,sales_team.group_sale_manager,1,1,1,1
 access_update_product_attribute_value_sale_manager,update.product.attribute.value,product.model_update_product_attribute_value,sales_team.group_sale_manager,1,1,1,0
```

Comment:
This one is meaningless to have if the user doesn't have the access to the product_attribute model, it can be removed too (and be given through the product_manager) group only